python_qt_binding/binding_helper.py

```diff
@@ -248,6 +252,8 @@ def createWidget(self, class_name, parent=None, name=''):
 
 def loadUi(uifile, baseinstance=None, custom_widgets=None):
     """
+    Load a provided UI file using PySide2.
```

Comment:
The reference of PySide2 is incorrect. This function uses the selected binding.

Reply:
Corrected